### DISCUSSION TITLE
App router example broken

### DISCUSSION THREAD

**Discussion by guifeliper** [amannn/next-intl#433] [General]
Hello guys, I have found that the example from here does not work. I notice this because I was trying to create a similar locale switch. 
https://next-intl-docs.vercel.app/examples/app-router

https://codesandbox.io/p/sandbox/github/amannn/next-intl/tree/main/examples/example-next-13?file=%2Fsrc%2Fapp%2F%5Blocale%5D%2Fpage.tsx%3A1%2C1
![2023-08-01 07 30 22](https://github.com/amannn/next-intl/assets/14844155/e238ee0f-7108-4cc9-81e6-79adca6d4266)

**Comment by aXenDeveloper** [comment]
Can you provide more information what is wrong? Maybe some example?

**Reply by guifeliper** [reply]
I have updated the discussion with a gif

**Reply by amannn** [reply]
Hmm, that's really odd. Can you reproduce the behavior locally? [The deployed version](https://next-intl-example-next-13.vercel.app/) works for me, maybe there's an issue with the CodeSandbox.

**Reply by amannn** [reply]
I think I found the issue, the example is running an outdated version of `next-intl` in CodeSandbox (unlike the Vercel deployment). Will have a fix published soon, thank you for the hint!

https://github.com/amannn/next-intl/pull/435
